Directory.Build.props

```diff
@@ -145,6 +145,7 @@
       win-x86;
       win-arm;
       win-arm64;
+      osx-arm64;
```

Comment:
A micro nit - the architectures for each platform seem to be listed in the following order: x64, x86, arm, arm64. It would be nice to keep that order for this newly added element. The same in the Dependencies.props below.